### PR TITLE
Portfolio: setup and mobile version skeleton PR

### PR DESCRIPTION
**Pull request by Iptysam**
Created an index.html file
Created a style.css file
Linters for html and css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,29 +9,31 @@
...
-    
-    </div>
-    <div class="images">
-        <div class="box">
-            <img src="C:\Users\SMT-LAB-Testing\My-Portfolio\img\Shape (2).png"/>
+        <ul class="img">
+            <li class="shape1"><img src="imgformobile/Shape(1).png" alt="shape"></li>
+            <li class="shape2"><img src="imgformobile/Shape.png" alt="shape"></li>
+        </ul>
+        <div class="navlinks">
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly put the hamburger in a nav tag

File: index.html [hunk 2]
```diff
@@ -9,29 +9,31 @@
...
     </div>
     <div class="container3">
-    <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </div>
     <div class="container4">
-        <p>Start collaboration</p>
+        <div class="info">
+            <p>Start collaboration</p>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly style the bottom to align with the hamburger.
- Your Design
Captureport5d
- Figma Design
Capturepor5

File: style.css [hunk 3]
```diff
@@ -1,116 +1,133 @@
...
+  left: 16px;
+  top: 70px;
+}
+
+.container2 h1 {
+  font-family: 'Playfair Display Bold', sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 110px;
+  font-size: 70px;
```
Comment (Kossi-stack) [inline style.css]: - [ ] Kindly fix the font size of your h1 tag according to the Figma template.

File: style.css [hunk 4]
```diff
@@ -1,116 +1,133 @@
...
   height: 26.47px;
-  left: 30.16px;
-  top: 0px;
+  left: 50px;
+  top: 0;
+}
 
-  font-family: 'Poppins';
+.info p {
+  font-family: 'Poppins Thin', sans-serif;
```
Comment (Kossi-stack) [inline style.css]: - [ ] Kindly fix the font family of your p tag according to the Figma template.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Iptysam,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ linter error checks validated
✅ Documented your work professionally
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: setup and mobile version skeleton PR" or "Portfolio: setup PR".
 - [ ] Kindly add [.gitignore](https://git-scm.com/docs/gitignore) file.
 - [ ] Kindly add [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.hintrc), [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.stylelintrc.json), to your project folder.
 - [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 - [ ] Kindly capitalize the first letter of your commit message in the next milestone so it looks professional.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 4 above)
Comment (Iptysam) [reply style.css]: when i use the font family as Poppins only i get an error in linters

File: index.html [hunk 5]
```diff
@@ -4,7 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name = "viewport" content="width=device-width, initial-scale=1">
-    <title>My Portfolio</title>
+    <title>Portfolio: setup and mobile version skeleton PR</title>
```
Comment (zhadier) [inline index.html]: - [ ] You seem to have misunderstood the previous reviewer, what he meant for you to change was the Pull Request title which you can see in the image below. The Project title is entirely up to you, but kindly change the PR title to something more descriptive 🌥️ 
image

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @Iptysam  👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🌤️ 



 ## Highlights

✔️ No linter errors
✔️ Github flow was set up correctly
✔️ Professional pull request title and description

 ## Required Changes ♻️
- [ ]  As the previous Reviewer requested, kindly put the hamburger in a `nav` tag instead of a `div` because it improves the semantic structure of the website since hamburger menus are usually used to navigate around the page.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
## Hi  @Iptysam , 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🍾💥
Congratulations! 🔥💯🚀🍄


## Highlights
 - No linters error .✅✅
 - Good GitHub flow ✅✅
 - Kudos on working on the requested changes.🔥🔥
 - Descriptive commit message.✅✅
 - Professional pull request description✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**